tests/Tests.elm

```diff
@@ -8,24 +8,30 @@ import Expect
 import Test exposing (..)
 
 
-set : EverySet Int
+type alias MyInt =
```

Comment:
I wasn't sure which of the two possibilities I should choose and went with the "more complex" one. I totally agree though that using a more realistic example is superior.